app/templates/navbar.html

```diff
@@ -24,11 +24,16 @@
         <a class="dropdown-item nav-link bg-dark" href="/new_zseries_recipe">New</a>
       </div>
     </li>
-    <li class="nav-item"><a class="nav-link" href="/brew_history">Brew History</a></li>
-    <li class="nav-item"><a class="nav-link" href="/ferm_history">Ferm History</a></li>
+    <li class="nav-item dropdown">
+      <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Sessions</a>
+      <div class="dropdown-menu dropdown-menu-right navbar-dark bg-dark" aria-labelledby="navbarDropdown">
+        <a class="dropdown-item nav-link bg-dark" href="/brew_history">Brew History</a>
+        <a class="dropdown-item nav-link bg-dark" href="/ferm_history">Ferm History</a>
+      </div>
+    </li>
     <li class="nav-item dropdown">
       <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">System</a>
-      <div class="dropdown-menu dropdown-menu-right navbar-dark bg-dark text-right" aria-labelledby="navbarDropdown">
+      <div class="dropdown-menu dropdown-menu-right navbar-dark bg-dark" aria-labelledby="navbarDropdown">

```

Comment:
every other navigation component was aligned left, so made this consistent

![image](https://user-images.githubusercontent.com/2158627/88352572-36ccd180-cd28-11ea-9422-91a6c6900be3.png)

Reply:
I explicitly made it right.. thought it looked goofy at the end not being right aligned but 🤷‍♂️

Reply:
Want them all right aligned? I'd be fine with that too 🤷‍♂️